Gemfile.lock

```diff
@@ -39,12 +39,12 @@ GEM
       tzinfo (~> 1.1)
     ansi (1.5.0)
     arel (6.0.3)
-    aws-sdk (2.2.6)
-      aws-sdk-resources (= 2.2.6)
-    aws-sdk-core (2.2.6)
+    aws-sdk (2.2.7)
+      aws-sdk-resources (= 2.2.7)
+    aws-sdk-core (2.2.7)
       jmespath (~> 1.0)
```

Comment:
We need to use aws-sdk v 1.6.0 in other way we became an error when we try to upload a picture.
I fix this in my PR but if yours pass first we became an error on heroku.